model/src/main/java/jetbrains/jetpad/model/collections/list/ObservableCollections.java

```diff
@@ -128,6 +129,69 @@ public void onEvent(CollectionItemEvent<? extends ItemT> event) {
     return EMPTY_LIST;
   }
 
+  private static <ItemT> ReadableProperty<Boolean> total(
```

Comment:
Such implementation leads to O(collection size) update for single collection change. 
What do you think of returning Property<Integer> with number of items that satisfy given predicate?